_source/logzio_collections/_tracing-sources/ecs-fargate.md

```diff
@@ -0,0 +1,138 @@
+Sending Metrics and Traces from ECS Fargate to Logz.io using AWS OTel Collector
+===============================================================================
+This document describes how to send metrics and traces from Amazon ECS Fargate to Logz.io using the AWS OTel Collector.
+**Only `awsecscontainermetrics` receiver metrics are collected by default.** 
+
+**Prerequisites:**
+
+*   An Amazon ECS Fargate cluster
+*   Applications instrumented with OpenTelemetry SDK (for traces)
+
+### Step 1: Deploy the CloudFormation template
+
+Click on the **Deploy to AWS** button below to deploy the CloudFormation template. This template will create the required resources and configurations for the AWS OTel Collector.
+
+[![Deploy to AWS](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://logzio-aws-integrations-us-east-1.s3.amazonaws.com/logzio-aws-ecs/ecs-fargate-collector.yaml)
+
+### Parameters
+
+The CloudFormation template requires the following parameters:
+
+*   `ClusterName`: The name of your ECS cluster from which you want to collect metrics.
+*   `CreateIAMRoles`: Specifies whether to create default IAM roles (True or False).
+*   `TaskRoleArn`: The role ARN you want to use as the ECS task role. (Optional, only required if you set `CreateIAMRoles` to False)
+*   `ExecutionRoleArn`: The role ARN you want to use as the ECS execution role. (Optional, only required if you set `CreateIAMRoles` to False)
+*   `SecurityGroups`: The list of SecurityGroupIds in your Virtual Private Cloud (VPC).
+*   `Subnets`: The list of Subnets in your Virtual Private Cloud (VPC).
+*   `LogzioTracingToken`: Your Logz.io tracing account token.
+*   `LogzioMetricsToken`: Your Logz.io metrics account token.
+*   `LogzioRegion`: Your Logz.io region. for example: `us`
+*   `LogzioListenerUrl`: Your Logz.io listener URL. for example: `https://listener.logz.io:8053`
+
+### Resources and Configuration
```

Comment:
Can we move the Resources and Configuration section after the steps? We need the instructions part to be as short as possible

Reply:
@Simplychee Please :)